clients/nexus-iq/src/main/kotlin/NexusIqService.kt

```diff
@@ -60,10 +60,18 @@ interface NexusIqService {
                         chain.proceed(authenticatedRequest)
                     }
                 }
+                addInterceptor { chain ->
+                    val request = chain.request()
+                    val csrfRequest = request.newBuilder()
+                        .header("X-CSRF-TOKEN", "ORT")
+                        .header("Cookie", "CLM-CSRF-TOKEN=ORT")
+                        .build()
+                    chain.proceed(csrfRequest)
+                }
```

Comment:
How about basically keeping this block, and conditionally also add `.header("Authorization", Credentials.basic(user, password))`, so we could manage with only one interceptor?